templates/.github/workflows/go-test.yml

```
@@ -42,7 +49,7 @@ jobs:
           # this means ./B's coverage will be significantly higher than 0%.
           run: go test -v -shuffle=on -coverprofile=module-coverage.txt -coverpkg=./... ./...
       - name: Run tests (32 bit)
-        if: ${{ matrix.os != 'macos' }} # can't run 32 bit tests on OSX.
+        if: ${{ matrix.os != 'macos' }} && env.SKIP32BIT == 'false' # can't run 32 bit tests on OSX.
```

Why is this not in the `${{ ... }}`? Is the env a string or a boolean?

---

The check is correct. We might need to adjust the templating though.

Either with:
```suggestion
        if: matrix.os != 'macos' && env.SKIP32BIT == 'false' # can't run 32 bit tests on OSX.
```
or: 
```suggestion
        if: ${{ matrix.os != 'macos' && env.SKIP32BIT == 'false' }} # can't run 32 bit tests on OSX.
```

Both will behave exactly the same.